### PR TITLE
Development helmchart

### PR DESCRIPTION
**Pull request by waleedhammam**
Helm chart for grid proxy server

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,45 @@
...
 RUN go mod download
 
 COPY . .
 RUN go build -o server cmds/proxy_server/main.go
 
+RUN wget https://github.com/threefoldtech/zinit/releases/download/v0.1/zinit -O /sbin/zinit \
+    && chmod +x /sbin/zinit
+
+# TODO: copy msgbus from release, currently locally
+COPY msgbus /usr/bin/msgbus
```
Comment (dmahmouali) [inline Dockerfile:28]: suggestion:
clone `rmb-go` repo in the builder and build it.


_the current stable branch in `rmb-go` repo is `testing`_

**Review by AhmedHanafy725 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (waleedhammam) [reply Dockerfile:28]: Instead we will import the package to be one binary: opened and issue for it here: https://github.com/threefoldtech/grid_proxy_server/issues/12

**Comment by waleedhammam** [comment]
Please don't merge until the base branch is complete

**Review by dmahmouali — APPROVED** [review]
:heart:

**Pull request merged**